tests/performance/load/groups/alerts.js

```diff
@@ -0,0 +1,40 @@
+import {Client, StatusOK} from 'k6/net/grpc';
+import {check, group} from 'k6';
+
+const GRPC_PORT = ":443";
+
+const client = new Client();
+
+client.load(['proto/stackrox', 'proto/googleapis'], 'api/v1/alert_service.proto');
+
+export function alertsGrpc(host, headers, baseTags, limit) {
+
+    host = host.replace('https://', '')
+    host = host.includes(':') ? host : host + GRPC_PORT;
+    client.connect(host);
+
+    let tags = { ...baseTags, limit: limit };
+
+    group('list alerts grpc', function () {
+        tags.limit = limit;
+        const params = {
+            metadata: headers, tags: t,
```

Comment:
I'm guessing it's a typo because I'm not able to find where `t` is defined.
```suggestion
            metadata: headers, tags: tags,
```